### PR TITLE
Add storage support for limit, include remove command, include undo command

### LINKED ISSUE #92: Feature: Remove (after list)

### PR DESCRIPTION
**Pull request by yetong1895**
Storage support is added for limit(load is working, save maybe)
added basic listing for limit I.E. "limits" to list.

**NEW**
Add **remove command** for entry,debt and limit.
I.e. in various mode, "remove 1" will remove the first log

Add **undo command** for entry and debt.
i.e. in various mode after a user input, **"undo"** will undo that input.

resolves #92, #89

### CODE REVIEW

**Review by tatayu — APPROVED** [review]

**Review by omupenguin — APPROVED** [review]

**Review by omupenguin — COMMENTED** [review]
Can add duke.txt to .gitignore?

**Review by Weng-Kexin — APPROVED** [review]

File: src/main/java/dolla/command/RemoveCommand.java [hunk 1]
```diff
@@ -1,18 +1,23 @@
...
  */
-public abstract class RemoveCommand extends Command {
-    protected String taskNumStr;
+public class RemoveCommand extends Command {
+    protected String logNumStr;
+    protected String mode;
 
-    public RemoveCommand(String taskNumStr) {
-        this.taskNumStr = taskNumStr;
+    public RemoveCommand(String mode, String taskNumStr) {
```
Comment (omupenguin) [inline src/main/java/dolla/command/RemoveCommand.java]: Let's not use 'taskNumStr' since that's the old system

**Review by omupenguin — CHANGES_REQUESTED** [review]
Refer to comments

**Comment by yetong1895** [comment]
> Can add duke.txt to .gitignore?

added

**Review by omupenguin — APPROVED** [review]

**Pull request merged**